Sources/ArcGISToolkit/Resources/ar.lproj/Localizable.strings

```diff
@@ -115,12 +126,19 @@ second being a corresponding cardinal direction (north, northeast,
 east, etc.). */
 "Compass, heading %lld degrees %@" = "البوصلة، تتجه %lld درجة/درجات %@";
 
+/* A label indicating a utility association's 
+containment is not visible. */
+"Content" = "بيت_Content_______________لاحقة";
```

Comment:
Translation for this pseudo string expected before July 29.

Reply:
Merged in [36db45b](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/pull/1221/commits/36db45b42fd5514c5f54a03a729477a5c42b8be4)